src/app/about/page.tsx

```diff
@@ -1,24 +1,28 @@
 export default function Index() {
   return (
     <>
-      <h1 className="my-16 md:mb-12 text-5xl md:text-8xl font-bold tracking-tighter leading-tight md:pr-8">
+      <h1 className="my-16 md:mb-6 text-5xl md:text-8xl font-bold tracking-tighter leading-tight md:pr-8">
       Our Mission
 	    </h1>
 	    <div>
-		  Fighting for affordable, accessible, and reliable public transit for the continually growing Dallas area.  
-	    </div>
-	    <h1>
-        About Us
+		    <h2 className="text-3xl mt-12 mb-4 leading-tight">
+        Fighting for affordable, accessible, and reliable public transit for the continually growing Dallas area.  
+	      </h2>
+      </div>
+	  <div>
+      <h1 className="my-8 md:mb-3 md:text-4xl font-bold leading-tight">
+      About Us
       </h1>
-      <div>
+    </div>
+      <div className="my-8 md:mb-6 text-2xl md:text-2xl leading-tight md:pr-8">
```

Comment:
Divs are misaligned. If you can, setup your IDE to auto format on save. I think I have the project setup to use ESLint

Reply:
fixed